src/gen/js/genTypes.ts

```diff
@@ -268,6 +268,7 @@ function renderTypeDoc(name: string, def: any): string[] {
   if (propLines.length) lines.push(`${DOC}`)
   join(lines, propLines)
   lines.push(' */')
+  lines.push(`const ${name}=null`)
```

Comment:
This is not really needed. For what i see the Types.js file is not really used anywhere.
It does not contain any code
It's there just to give code completion.
My problem was that some CodeEditor's would not find the `@typedef` and won't give code completion as they assume that typedef is not defined.
But this probably a problem to be solved in code editors, not in code gen, thought it does not hurt anyone